services/activitylog/pkg/service/response.go

```diff
@@ -20,16 +20,19 @@ import (
 
 // Translations
 var (
-	MessageResourceCreated = l10n.Template("{user} added {resource} to {space}")
-	MessageResourceTrashed = l10n.Template("{user} deleted {resource} from {space}")
-	MessageResourceMoved   = l10n.Template("{user} moved {resource} to {space}")
-	MessageResourceRenamed = l10n.Template("{user} renamed {oldResource} to {resource}")
-	MessageShareCreated    = l10n.Template("{user} shared {resource} with {sharee}")
-	MessageShareDeleted    = l10n.Template("{user} removed {sharee} from {resource}")
-	MessageLinkCreated     = l10n.Template("{user} shared {resource} via link")
-	MessageLinkDeleted     = l10n.Template("{user} removed link to {resource}")
-	MessageSpaceShared     = l10n.Template("{user} added {sharee} as member of {space}")
-	MessageSpaceUnshared   = l10n.Template("{user} removed {sharee} from {space}")
+	MessageResourceInSpaceCreated  = l10n.Template("{user} added {resource} to {space}")
+	MessageResourceInSpaceTrashed  = l10n.Template("{user} deleted {resource} from {space}")
+	MessageResourceInSpaceMoved    = l10n.Template("{user} moved {resource} to {space}")
```

Comment:
I don't think we need these at all now. Apparently there was a misunderstanding with translation text. I'd recommend to use `{folder}` everywhere.